test/controllers/events_controller_test.rb

```diff
@@ -0,0 +1,48 @@
+require 'test_helper'
+
+class EventsControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @event = events(:one)
+  end
+
+  test 'should get index' do
+    get events_url
+    assert_response :success
+  end
+
+  test 'should get new' do
+    get new_event_url
+    assert_response :success
+  end
+
+  test 'should create event' do
```

Comment:
- [ ] Please include instructions on how to run your test in the README. I couldn't run these tests.
- [ ] I encourage you to use rspec for testing.
- [ ] Models should have unit tests for validations and associations (Rspec).
- [ ] Authentication should have integration tests (Rspec + Capybara).
- [ ] Event management features should have integration tests (Rspec + Capybara).